bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -157,9 +158,44 @@ public void dispose() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no commands
-    }
+        handleCommand(channelUID.getId(), command);
+    }    
+    
+    public void handleCommand(String channel, Command command) {
```

Comment:
Why don't you put that in the method above?

Reply:
The new added CLIP (Connected Lighting Internet Protocol) sensor can be set from CLIP RESTful API when received a command from LAN or internet. 
I will add handleCommand for LightLevelHandler/PresenceHandler/TemperatureHandler later for supporting CLIP.

The reason for put them in single place is:

The actions of CLIP or ZLL sensor on HUE hub are basiclly same.
The codes between CLIP sensors and ZLL Sensors are essentially same. 
The codes of handleCommand of different sensors are essentially same. 
The status can have two or more states like ZLL/CLIPTemperature sensor may have temperature and humidity in single sensor.  

So, Let's look at 8 types of sensors:

1/2：ClipHandler (CLIPGenericFlag & CLIPGenericStatus) - flag/status, 
3/4：LightLevelHandler (ZLLLightlevel & CLIPLightlevel) - lightlevel, 
5/6：PresenceHandler (ZLLPresence & CLIPPresence) - presence, 
7/8：TemperatureHandler (ZLLTemperature & CLIPTemperature) - temperature/humidity, 
... more other sensor...

I think put all the code in single place will be easy to write and extend.  
What's your suggestion?